lib/mix/tasks/ecto.migrate.ex

```diff
@@ -65,7 +65,8 @@ defmodule Mix.Tasks.Ecto.Migrate do
 
     {opts, _, _} = OptionParser.parse args,
       switches: [all: :boolean, step: :integer, to: :integer, quiet: :boolean,
-                 prefix: :string, pool_size: :integer, log_sql: :boolean],
+                 prefix: :string, pool_size: :integer, log_sql: :boolean,
+                 strict_version_order: :boolean],
```

Comment:
Can you please update the docs for this module to mention the --struct-version-order switch?